platform/mellanox/docker-syncd-mlnx/sai-common.profile

```diff
@@ -0,0 +1,3 @@
+SAI_DUMP_STORE_PATH=/var/log/mellanox/sdk-dumps
+SAI_DUMP_STORE_AMOUNT=10
+SAI_DUMP_MFT_CFG_PATH=/etc/mft/fwtrace_cfg
```

Comment:
@noaOrMlnx , I see two kind of paths. One is the one in the default profile and the other appear on several SKUs with the path of:
SAI_DUMP_MFT_CFG_PATH=/etc/sonic/mft/fwtrace_cfg

Please check if this is a mistake and all should point to the same path